Resources/config/services.yml

```diff
@@ -4,6 +4,9 @@ services:
         autoconfigure: true
         public: false
 
+        bind:
+            $kernelBundles: '%kernel.bundles%'
+
```

Comment:
Do these parameters get exposed beyond the bundle boundaries, and can they potentially conflict with user defined binds? If so, we should rather pass them to the event listeners directly - if not, then this is fine 🙂